src/js/main.js

```diff
@@ -363,6 +363,22 @@ if (document.body.dataset.page === "integrations") {
   }
 }
 
+// Add underline class to menu links on Mobile
+
+function setActiveMenuItem() {
+  const currentPath = window.location.pathname;
+  const menuItems = document.querySelectorAll("nav a");
+
+  menuItems.forEach((item) => {
+    const itemPathname = item.pathname;
+    const shouldBeUnderlined = currentPath.startsWith(itemPathname);
+
+    item.classList.toggle("border-b-2 border-gray-600", shouldBeUnderlined);
```

Comment:
Ah toggle deals with a single class at a time!

```suggestion
    item.classList.toggle("border-b-2", shouldBeUnderlined);
    item.classList.toggle("border-gray-600", shouldBeUnderlined);
```